### ISSUE TITLE
Compilation error on Imagineer oder than 0.1.0

### ISSUE THREAD

**Issue by jungrafael** [SenecaSystems/gutenex#7]
```
== Compilation error on file lib/gutenex/pdf/images.ex ==
** (CompileError) lib/gutenex/pdf/images.ex:5: Imagineer.Image.__struct__/0 is undefined, cannot expand struct Imagineer.Image
    (elixir) src/elixir_map.erl:58: :elixir_map.translate_struct/4
    (stdlib) lists.erl:1353: :lists.mapfoldl/3
    (stdlib) lists.erl:1354: :lists.mapfoldl/3
```

**Comment by Devalo** [comment]
Hello. I'm getting the same error as above when running the example.
imagineer v 0.2.1

**Comment by jungrafael** [comment]
This is a compatibility bug between gutenex and the latest versions of imagineer. You need to limit the imagineer version to 0.1.0.
See [https://github.com/jungrafael/gutenex](https://github.com/jungrafael/gutenex).

**Comment by tyre** [comment]
Apologies, haven't gotten to fixing this. The newest Imagineer was rewritten to handle parsing and writing of images. I broke API compatibility.

**Comment by tyre** [comment]
@jungrafael do you have a commit that fixes this? Happy to accept a PR as a stopgap

**Comment by tyre** [comment]
Nvm, fixed this in `0.2.0` which is now on hex
